pkg/meta/redis.go

```diff
@@ -2824,6 +2824,32 @@ func (m *redisMeta) ListSlices(ctx Context, slices map[Ino][]Slice, delete bool,
 	return errno(err)
 }
 
+func (m *redisMeta) RepairInode(ctx Context, parent, inode Ino) syscall.Errno {
```

Comment:
```
Check(ctx Context, inode Ino, repair bool) syscall.Errno
```

Reply:
It's kind of difficult to repair an inode directly.
For example, if the inodeKey is lost, we need parent inode & type information to recover it.